Library/Formula/uwsgi.rb

```diff
@@ -45,6 +45,10 @@ class Uwsgi < Formula
   depends_on "v8" => :optional
   depends_on "zeromq" => :optional
 
+  # Fix build on case-sensitive filesystems
+  # https://github.com/Homebrew/homebrew/issues/45560
+  inreplace "plugins/alarm_speech/uwsgiplugin.py", "'-framework appkit'", "'-framework AppKit'"
```

Comment:
Submitted to upstream and merged in head already. These folks don't dawdle.

Added a link to the upstream PR.